### DISCUSSION TITLE
Mermaid in Project Readme

### DISCUSSION THREAD

**Discussion by morigs** [community/community#37602] [Other Feature Feedback, Questions, & Ideas]
Mermaid is not rendered in Project readme. For instance, the following md

<pre>
```mermaid
  graph TD;
      A-->B;
      A-->C;
      B-->D;
      C-->D;
```
</pre>

looks like this in preview

![image](https://user-images.githubusercontent.com/13474134/198522314-947600f3-16f6-4753-acb8-04d1d17ac75e.png)

**Comment by jfullstackdev** [comment]
I just pasted it, it's working

![image](https://user-images.githubusercontent.com/47092464/198614101-f27bb8dc-fa86-43b9-80b3-bad0ea19a3c2.png)

**Reply by morigs** [reply]
Perhaps you tried `README.md` in a repo. I'm talking about Perojects (beta) readme field

**Comment by mtchoy** [comment]
I also noticed that the Mermaid diagram doesn't render on the Projects Item page.

**Comment by h14h** [comment]
I've seen that mermaid diagrams render properly in `.md` files stored in git, as well as in the descriptions of GitHub Issues. The only place I've seen them _not_ render is the README GitHub Projects.

Screenshot taken on January 31st, 2024:
<img width="1411" alt="Screenshot 2024-01-31 at 11 14 36 AM" src="https://github.com/community/community/assets/121139610/1fc41239-d790-408e-b460-7373c913f704">


Here is the markdown in question:

````
# Testing Mermaid

```mermaid
sequenceDiagram
    Alice->>John: Hello John, how are you?
    John-->>Alice: Great!
    Alice-)John: See you later!
```
````

...which in most places renders like so:

# Testing Mermaid

```mermaid
sequenceDiagram
    Alice->>John: Hello John, how are you?
    John-->>Alice: Great!
    Alice-)John: See you later!
```
